meinberlin/react/contrib/ControlBarSearch.jsx

```diff
@@ -27,29 +29,18 @@ export const ControlBarSearch = (props) => {
       onSubmit={handleSubmit}
       className="input-group"
     >
-      <label
-        htmlFor="id-filter-search"
-        className="visually-hidden"
-      >
-        {translated.searchFor}
-      </label>
-      <input
-        className="form-control"
-        type="search"
-        id="id-filter-search"
-        placeholder={translated.searchFor}
+      <Input
+        label={translated.search} id="id-filter-search"
         onChange={handleChange}
-        value={value}
-      />
-      <button
-        className="btn btn--light input-group__after"
-        type="submit"
+        before={<i className="fa fa-search" aria-hidden="true" />}
       >
-        <i className="fa fa-search" aria-hidden="true" />
-        <span className="visually-hidden">
-          {translated.startSearch}
-        </span>
-      </button>
+        <button type="submit" title={translated.startSearch}>
+          <i className="fa fa-arrow-right" aria-hidden="true" />
+          <span className="visually-hidden">
+            {translated.startSearch}
+          </span>
+        </button>
+      </Input>
```

Comment:
I think we should stick to the bo implementation as much as possible for the search as I don't think we had any design changes? also there are other searches on the site so they should all look as much the same as possible. I think main change here is just the icons, will check with Marcelo as they were using font awesome but recently changed to these bicon ones.

Reply:
check `Searchform` in the styleguide